gn3/computations/biweight.R

```diff
@@ -0,0 +1,23 @@
+
+library(WGCNA)
+
+arg_values <- commandArgs(trailingOnly = TRUE)
```

Comment:
@Alexanderlacuna, @zsloan , @pjotrp and @robwwilliams how about we stick to one style guide for R code. I propose: https://web.stanford.edu/class/cs109l/unrestricted/resources/google-style.html